pkg/timeutil/timeutil.go

```diff
@@ -20,9 +24,32 @@ func Date(datetime time.Time) time.Time {
 func LastWeekday(weekday time.Weekday) time.Time {
 	date := Date(time.Now())
 
-	for date.Weekday() != weekday {
+	for date.Weekday() !=  weekday{
```

Comment:
this looks like an extra space here. something running the formatting tool would pick up I think!